ONSdigital/eq-author-app
# Fix latestDate and change mix/max wording

### What is the context of this PR?

Updates the text for duration validation on Date ranges.

I also noticed the latest date validation wasn't displaying the correct text so I changed that too

### How to review 

> Add to the list below as appropriate, including screenshots when necessary

1. Create a new questionnaire with a date range question
2. Add validation for both earliest/latest date and duration
3. Duration text should be as described in the ticket
4. Latest date should say less instead of more

The codecov diff is covered in the duration testing. Didn't want to add a conflicting test
